### ISSUE TITLE
Allow `plot_cap()` to show predictions at the observation level

### ISSUE THREAD

**Issue by tomicapretto** [bambinos/bambi#623]
`plot_cap()` show the adjusted mean. After #622 it will be able to show other adjusted parameters. But it doesn't show the adjusted posterior predictive distribution, and it would be really nice to have that.

**Comment by tomicapretto** [comment]
Closed with #668
